### ISSUE TITLE
stringField regex supported?

### ISSUE THREAD

**Issue by PhoenixAndMachine** [stephrdev/django-mongoforms#23]
I made a ModelForm by metaclass a document, but in which the stringField seems not to support regex in model.
with error such as:

Error when calling the metaclass bases
    **init**() got an unexpected keyword argument 'regex'

**Comment by lig** [comment]
Regexes are supported for Document StringFields in django-mongoforms.

Could you please provide exact code of your form class containing form class definition and string field definition in the form and in the document class.

**Comment by PhoenixAndMachine** [comment]
models.py

```
from mongoengine import *
from django.utils.translation import ugettext_lazy as _
from mongoengine.django.auth import *
# Create your models here.

class Customer(User):
    email = EmailField(verbose_name=_("email"))
    phone_number = StringField(regex='\d{11}', verbose_name=_("phone"))
    addresses = ListField(StringField(max_length=1024), verbose_name=_("addresses"))
    birthday = DateTimeField(verbose_name=_("birthday"))
```

forms.py

```
class CustomerRegisterLoginForm(MongoForm):
    class Meta:
        document = Customer
        fields = ['phone_number']
```

views.py

```
def create_user(request):
    customer_register_form = CustomerRegisterLoginForm()
    print customer_register_form
    context = RequestContext(request, {'customer_register_form': customer_register_form})
    return render_to_response('create_user.html', context)

```

**Comment by lig** [comment]
@PhoenixGrey0108 Thank you!

I will look into it shortly.

**Comment by lig** [comment]
Found the bug cause. Going to fix.

**Comment by lig** [comment]
Fix committed into `issue_23` branch https://github.com/stephrdev/django-mongoforms/commit/03c940f4409f02d45d5a2b7574902e58ebd90b57

**Comment by lig** [comment]
Fixed. Fix released in v0.3.1. Uploaded to pypi.
